File: app/src/main/java/com/example/simonastojanovic/memeapp/model/MemesItem.java
package com.example.simonastojanovic.memeapp.model;

import com.google.gson.annotations.SerializedName;

public class MemesItem{

	@SerializedName("name")
	private String name;

	@SerializedName("width")
	private int width;

	@SerializedName("id")
	private String id;

	@SerializedName("url")
	private String url;

	@SerializedName("height")
	private int height;

	public void setName(String name){
		this.name = name;
	}

	public String getName(){
		return name;
	}

	public void setWidth(int width){
		this.width = width;
	}

	public int getWidth(){
		return width;
	}

	public void setId(String id){
		this.id = id;
	}

	public String getId(){
		return id;
	}

	public void setUrl(String url){
		this.url = url;
	}

	public String getUrl(){
		return url;
	}

	public void setHeight(int height){
		this.height = height;
	}

	public int getHeight(){
		return height;
	}

	@Override
 	public String toString(){
		return 
			"MemesItem{" + 
			"name = '" + name + '\'' + 
			",width = '" + width + '\'' + 
			",id = '" + id + '\'' + 
			",url = '" + url + '\'' + 
			",height = '" + height + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/example/simonastojanovic/memeapp/model/MemesItem.kt
package com.example.simonastojanovic.memeapp.model

import com.google.gson.annotations.SerializedName

class MemesItem {

    @SerializedName("name")
    var name: String? = null

    @SerializedName("width")
    var width: Int = 0

    @SerializedName("id")
    var id: String? = null

    @SerializedName("url")
    var url: String? = null

    @SerializedName("height")
    var height: Int = 0

    override fun toString(): String {
        return "MemesItem{" +
                "name = '" + name + "\'" +
                ",width = '" + width + '\''.toString() +
                ",id = '" + id + '\''.toString() +
                ",url = '" + url + '\''.toString() +
                ",height = '" + height + '\''.toString() +
                "}"
    }
}